package.json

```diff
@@ -33,10 +33,10 @@
     "typescript": "^4.9.4",
     "yaml": "^2.2.0"
   },
+  "bundleDependencies": [
+    "tests"
+  ],
```

Comment:
They accept both `bundle` and `bundled`, but the former is recommended. Either way works